listenbrainz/webserver/static/css/recommendation-page.less

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
maybe we can have a common.less for common styling. Moreover there are quite a few you dont need.

Reply:
I wanted to have a separate less file for recs. It will be easier to only change rec components and styling in the future. 

> Moreover there are quite a few you dont need.

Being more specific will help.

Reply:
I think if you nuke out the styling for the redundant media query component then it will serve the purpose. I can be specific if you are still stuck

Reply:
ok I will see to it